ddtrace/tracer.py

```diff
@@ -76,7 +79,8 @@ def context_provider(self):
         return self._context_provider
 
     def configure(self, enabled=None, hostname=None, port=None, sampler=None,
-                context_provider=None, wrap_executor=None, settings=None):
+                  context_provider=None, wrap_executor=None, priority_sampling=None,
```

Comment:
By default, priority sampling is disabled, it needs to be turned on explicitly by setting `priority_sampling` to `True`.